docs/nut-names.txt

```diff
@@ -534,6 +536,7 @@ battery: Any battery details
                                  to "Warning" state (percent)        | 50
 | battery.charger.status       | Status of the battery charger
                                  (see the note below)                | charging
+| battery.charger.type         | ABM charger type                    | ABM
```

Comment:
I think this description can be generalized to "Type of battery charger (e.g. ABM)" which can eventually grow to other values as drivers learn to supply them. Same for `cmdvartab` then.